src/_includes/banner.html

```diff
@@ -1,6 +1,10 @@
 {% if page.show_banner -%}
   <div class="site-banner site-banner--default" role="alert">
       Flutter 1.9 正式发布！查看
-      <a href="/posts/flutter-news-from-gdd-china-flutter1.9.html" target="_blank">发布内容</a>。
```

Comment:
Please revert to my version for this file since the site was played a the community site role, but all these branding permission were granted to Google Inc. In other words, we won't show the Showcase page in .cn site, thanks!